src/Uno.UI/UI/Xaml/UIElement.netstdref.cs

```diff
@@ -11,6 +11,7 @@ public partial class UIElement : DependencyObject
 	{
 		public UIElement()
 		{
+			_isFrameworkElement = this is FrameworkElement; // Avoids unused field error
```

Comment:
Codacy found an issue: [Offload the code that's conditional on this 'is' test to the appropriate subclass and remove the test.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5975891)